README.md

```diff
@@ -24,6 +24,9 @@ Dependencies:
 
 #### Homebrew
 
+can install homebrew from the below link
+"https://docs.brew.sh/Installation"
```

Comment:
It's a good idea to link to Homebrew. However, I don't think we need to add an additional sentence like this. How about turning “Homebrew” in the existing sentence into a link?

```
`hub` can be installed through [Homebrew](https://brew.sh) on macOS:
```